session/portforward.go

```diff
@@ -35,7 +35,9 @@ const (
 )
 
 var (
-	errPortForwardInvalidMessage     = errors.New("invalid port-forward message: missing connection_id, remort_port or protocol")
+	errPortForwardInvalidMessage = errors.New(
+		"invalid port-forward message: missing connection_id, remort_port or protocol",
```

Comment:
as long as we are changing this line, perhaps "remort_port"->`remote_port` ?

Reply:
Good catch